CHANGES.md

```diff
@@ -8,6 +8,7 @@ Fixed Issues:
 * [#2721](https://github.com/ckeditor/ckeditor-dev/issues/2721): Fixed: Sublist items are reversed when higher level list item is removed.
 * [#2527](https://github.com/ckeditor/ckeditor-dev/issues/2527): Fixed: [Emoji](https://ckeditor.com/cke4/addon/emoji) autocomplete order now prioritize emojis with name started from used string.
 * [#2572](https://github.com/ckeditor/ckeditor-dev/issues/2572): Fixed: Icons in [Emoji](https://ckeditor.com/cke4/addon/emoji) dropdown navigation groups are not centered.
+* [#2208](https://github.com/ckeditor/ckeditor-dev/issues/2208): [IE8-11] Fixed: [Autolink](https://ckeditor.com/cke4/addon/autolink) plugin duplicated with native browser implementation.
```

Comment:
I think it's redundant to list IE versions when it applies to every supported version. Tag `[IE]` was used 27 times in changelog.
```suggestion
* [#2208](https://github.com/ckeditor/ckeditor-dev/issues/2208): [IE] Fixed: [Autolink](https://ckeditor.com/cke4/addon/autolink) plugin duplicated with native browser implementation.
```